packaging/bundle/Dockerfile

```diff
@@ -23,13 +23,13 @@ ARG DOCKER_REPO=docker.io
 
 
 ######## Base image for subsequent stages ########
-FROM ${DOCKER_REPO}/ubuntu:24.04 AS base
+FROM ${DOCKER_REPO}/python:${PYTHON_VERSION} AS base
```

Comment:
Do you know what is the python image based on? given the cmds are same it should be ubuntu or debian, but am wondering which one are using now?

Reply:
Let give me it a spin